crates/nu-protocol/src/module.rs

```diff
@@ -167,7 +171,7 @@ impl Module {
                 vec![]
             } else {
                 vec![(
-                    final_name.clone(),
+                    name_override.unwrap_or(&self.const_name).to_vec(),
```

Comment:
I don't think it's necessary to save `const_name` to a module.
We can just `normalize_variable_name` for final_name.

Reply:
TBH I can't find a good place either, I think it's ok to make a duplicate function in `module.rs`.